FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt

```diff
@@ -30,6 +30,7 @@ libraries:
   + https://www.FreeRTOS.org/jobs/
   + https://www.FreeRTOS.org/ota/
   + https://www.FreeRTOS.org/pkcs11/
+  + https://www.FreeRTOS.org/mqtt_lts/
```

Comment:
Did we need it here? This seems to contain the demo for Beta1 libraries only

Reply:
Good catch, I guess this what happens when the directories differ by 1 character